taas/config.yml

```diff
@@ -0,0 +1,44 @@
+---
+# TODO: We should have a config.d or similar setup; this means adding
+#       additional services doesn't result in them fighting for control of the
+#       same config file.
+
+sources:
+    # This maps directly to the API URL
+    functional_roles:
+        v1:
+            # Extracted from the spreadsheet URL
```

Comment:
@emmajane : I like the idea of including the full URL. It's more human friendly and easier to understand all around.

However I believe that's also orthogonal to this PR, so I've opened [TAAS-32](https://humanitarian.atlassian.net/browse/TAAS-32) with that request.